src/native/components/Lotes/LotesCarousel.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Image, View } from 'react-native';
+import Carousel from 'react-native-looped-carousel-improved';
+
+import Spacer from '../Spacer';
+
+import CarouselItem from './CarouselItem';
+
+const LotesCarousel = ({
```

Comment:
How different will this carousel be from other ones? Could we have a general carousel component that is flexible for this usage?

Reply:
probably, but i'm not sure.  we can try implementing this first when doing the portfolio. thats the only other place with a carousel